src/structures/Contact.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Base = require('./Base');
+
+/**
+ * Represents a Contact on WhatsApp
+ * @extends {Base}
+ */
+class Contact extends Base {
+    constructor(client, data) {
+        super(client);
+
+        if(data) this._patch(data);
+    }
+
+    _patch(data) {
+        this.id = data.id;
+        this.isBusiness = data.isBusiness;
+        this.isEnterprise = data.isEnterprise;
+        this.labels = data.labels;
+        this.name = data.name;
+        this.pushname = data.pushname;
+        this.sectionHeader = data.sectionHeader;
+        this.shortName = data.shortName;
+        this.statusMute = data.statusMute;
+        this.type = data.type;
+        this.verifiedLevel = data.verifiedLevel;
+        this.verifiedName = data.verifiedName;
+
+        return super._patch(data);
+    }
+    
+    /**
+     * Gets the Profile Image of the User
+     */
+    async getProfileImageUrl() {
```

Comment:
This doesn't seem to be working. Two issues I see here:

1. Store.ProfilePicThumb seems to be lazily loaded in some way, which could cause the `get` to not return anything. 
2. When it does find it, I'm not getting back a profile image url, but rather simply an object with the contact ID.

On another note, `Contact` has a `profilePicThumb` property, so we can probably load this in when it's serialized with `WWebJS.getContactModel`, removing the need to use an additional async function here to get the profile image

Reply:
I took another look at this and Contact.profilePicThumb isn't reliable either. They can potentially not be set. However, I found another module we can use:

`mR.findModule("profilePicFind")[0]`
 
It has a couple functions related to getting and setting profile images. With the following we can ensure to always get the profile image URL for any WhatsApp user. The only caveat is if the profile has their profile picture privacy settings set to ignore the current user, it will return a 401.

Reply:
I rechecked... Seems only to work with Contacts that are on the screen.
For now i can remove the profilepicThumb and it can be integrated later on?

Reply:
Yeah that sounds ok. I have an idea for how to do it so I could take a crack at it.